sidebars.js

```diff
@@ -49,7 +49,28 @@ module.exports = {
             "autographa/How-To/Text-Translate-Mode/p2-4-12-notifications",
```

Comment:
The pages/md files referenced here from line 42 - 49 have been removed from the repo. Remove the non-existing file references. They are listed below.

```
           "autographa/How-To/Text-Translate-Mode/p2-4-5-split-column",
            "autographa/How-To/Text-Translate-Mode/p2-4-6-lock-unlock-editor",
            "autographa/How-To/Text-Translate-Mode/p2-4-7-create-bookmarks",
            "autographa/How-To/Text-Translate-Mode/p2-4-8-file-menu",
            "autographa/How-To/Text-Translate-Mode/p2-4-9-edit",
            "autographa/How-To/Text-Translate-Mode/p2-4-10-about",
            "autographa/How-To/Text-Translate-Mode/p2-4-11-font",
            "autographa/How-To/Text-Translate-Mode/p2-4-12-notifications",
```